### PR TITLE
Use motor name as urdf-joint name

### PR DESCRIPTION
**Pull request by Simon-Steinmann**
Converting a urdf to .proto, we use the jointname as the motorname. Converting .proto to .urdf should do the same in reverse, not take the sensor name.

### CODE REVIEW

**Comment by lukicdarkoo** [comment]
This will break `JointStatePublisher`:
https://github.com/cyberbotics/webots_ros2/blob/master/webots_ros2_core/webots_ros2_core/joint_state_publisher.py

`JointStatePublisher` has to be improved to retrive corresponding `PositionSensor` (with `wb_motor_get_position_sensor()`).

**Comment by Simon-Steinmann** [comment]
I created a PR for ROS2 adressing this issue https://github.com/cyberbotics/webots_ros2/pull/131

File: src/webots/nodes/WbJoint.cpp [hunk 1]
```diff
@@ -296,8 +296,8 @@ void WbJoint::updateJointAxisRepresentation() {
 }
 
 const QString WbJoint::urdfName() const {
-  if (positionSensor() && positionSensor()->findSFString("name"))
-    return getUrdfPrefix() + positionSensor()->findSFString("name")->value();
+  if (motor() && motor()->findSFString("name"))
+    return getUrdfPrefix() + motor()->motor("name")->value();
```
Comment (DavidMansolino) [inline src/webots/nodes/WbJoint.cpp:300]:
```suggestion
  if (motor() && motor()->findSFString("name"))
    return getUrdfPrefix() + motor()->motor("name")->value();
  else if (positionSensor() && positionSensor()->findSFString("name"))
    return getUrdfPrefix() + positionSensor()->findSFString("name")->value();
```

Just to handle the case of a non-actuated joint.

**Comment by DavidMansolino** [comment]
This file probably needs to be updated: https://github.com/cyberbotics/webots/blob/master/tests/api/controllers/robot_urdf/reference_robot.urdf

**Review by DavidMansolino — APPROVED** [review]
Looks all good now.

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**